modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java

```diff
@@ -250,6 +395,18 @@ protected ObjectField testPatchObjectField_addObjectField()
 		return _addObjectField();
 	}
 
+	@Override
+	protected ObjectField
+			testPostObjectDefinitionByExternalReferenceCodeObjectDefinitionExternalReferenceCodeObjectField_addObjectField(
+				ObjectField objectField)
+		throws Exception {
+
+		return objectFieldResource.
+			postObjectDefinitionByExternalReferenceCodeObjectDefinitionExternalReferenceCodeObjectField(
+				testGetObjectDefinitionByExternalReferenceCodeObjectDefinitionExternalReferenceCodeObjectFieldsPage_getObjectDefinitionExternalReferenceCode(),
```

Comment:
@p-albuquerque can we pass the `_objectDefinition.getExternalReferenceCode()` instead?